config/archives/linuxcnc-uspace.list.binary

```diff
@@ -0,0 +1 @@
+deb [arch=amd64] http://buildbot2.highlab.com/debian/ bookworm 2.9-uspace
```

Comment:
Why do you limit the arch here?

What about specifying the key to use like done with the repo added in config/hooks/normal/0520-Qtpyvcp-repos.hook.chroot? It would reduce the attack surface a bit.

Reply:
Currently, only amd64 is present on the new repo. When doing an apt update, without the arch flag, warnings are raised that i386 repos cannot be found. On review, this could be because of a malformed linuxcnc repository but I am out of my depth
 http://buildbot2.highlab.com/debian/dists/bookworm/2.9-uspace/ 

I did try to use the new apt sources format but got errors so reverted.